<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="LAPTOP-CHFSUI5B" failures="1" tests="2" name="Default test(failed)" time="0.311" errors="0" timestamp="2020-05-29T14:57:14 IST">
  <testcase classname="testNG.ReRunFailedTestCases" name="testfail" time="0.224">
    <failure type="org.testng.internal.thread.ThreadTimeoutException" message="Method testNG.ReRunFailedTestCases.testfail() didn&amp;apos;t finish within the time-out 200">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method testNG.ReRunFailedTestCases.testfail() didn't finish within the time-out 200
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- testfail -->
  <testcase classname="testNG.ReRunFailedTestCases" name="testcase1" time="0.007"/>
  <testcase classname="testNG.ReRunFailedTestCases" name="testing" time="0.0">
    <ignored/>
  </testcase> <!-- testing -->
</testsuite> <!-- Default test(failed) -->
